osu.Game/Rulesets/Judgements/DefaultJudgementPiece.cs

```diff
@@ -38,18 +38,16 @@ public DefaultJudgementPiece(HitResult result)
         /// </remarks>
         public virtual void PlayAnimation()
         {
-            switch (Result)
+            if (Result != HitResult.None && !Result.IsHit())
```

Comment:
It's a bit haphazard because I was just matching the existing implementations (to avoid figuring out what needs to actually when for `HitResult.None` if anything...) but I'm not against standardising them. Just takes a bit more effort.

Reply:
I've made some changes.. see what you think.